### ISSUE TITLE
Only scans up to 30 ip addresses

### ISSUE THREAD

**Issue by stefan-imp** [joshmarinacci/razzmaster#3]
For some reason scan is only checking the first 30 ip addresses on the subnet (x.x.x.1 - x.x.x.30).  My Pi is at (x.x.x.55), so is not found.

If I run `razzmaster info --host x.x.x.55` the output looks correct.

I'm running ubuntu 14.04 on my PC.

**Comment by stefan-imp** [comment]
Ah, it looks like this is the same as issue #2.  I tried editing find.js as suggested.  Scanning up to 100 was successful.  Scanning to 200 was not, it crashed with too many processes.

**Comment by joshmarinacci** [comment]
Yep. I need to rewrite this section. I can't just up it to 255 because of process limits so it will need to go in batches.
